docs/development/01-contribution.md

```diff
@@ -0,0 +1,59 @@
+
+In addition you should:
+
+
+- Explain any deliverables that have already been achieved
+- Outline what needs to be done for the proposal to be completed
+
+The proposal should then be discussed with  the community (including, for example, the KILT Technical Committee, governance, or relevant project team) using community channels such as [Discord](https://discord.gg/7uyfMXh6AT). If the community is not in agreement with the proposal, it is unlikely that it would be approved by governance. See the [guide to creating a proposal](https://www.kilt.io/treasury/create-a-proposal/).
+
+Governance approval for a Treasury spend will also  includes list of the deliverables as outlined in the proposal.
+
+## Bug Reports
+
+We try our best, but bugs are an everyday reality with all software and are bound to happen. We can't fix bugs we don't notice, so your input if you spot something gives us the best possibility to keep the project running smoothly and securely.
+
+If you are unsure that a bug is a bug, it is best to open an issue and report it anyway. The active developers will evaluate it and help to figure out the issue.
+
+It is helpful to check if a report has already been filed in the related project. Search the issues board for possible phrases that match the description of the bug. It's possible you may not find an issue, but it's better to file a duplicated bug than not report one.
+
+Once you begin reporting the bug, write a descriptive title so that if others find the same issue they can either add to your findings or know that the bug has already been reported. A bug report should be as detailed as possible, including steps to reproduce, screenshots, error reports, or code snippets. The more details you provide, the easier it is to fix the issue.
+
+## Pull Requests
+
+Pull Requests (PR) are an integral part of contributions to change KILT. GitHub itself has some [excellent documentation](https://help.github.com/articles/about-pull-requests/) on using the Pull Request feature. KILT uses the "[fork and pull](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/about-collaborative-development-models)" model described here, where contributors push changes to their personal fork and create pull requests to bring those changes into the source repository.
+
+Before starting a PR, it’s best to contact other active developers and discuss the proposed changes. Open an issue or directly contact some of the devs on [Discord](https://discord.gg/7uyfMXh6AT) to begin discussing and making the proposal. Once approved, contributors can open a PR for review. The PR will be reviewed and, if accepted, merged into the corresponding repository.
```

Comment:
```suggestion
Before starting a PR, it is best to contact other active developers and discuss the proposed changes. Open an issue or directly contact some of the developers on [Discord](https://discord.gg/7uyfMXh6AT) to begin discussing and making the proposal. Once approved, contributors can open a PR for review. The PR will be reviewed and, if accepted, merged into the corresponding repository.
```

Reply:
We could also add mentioning of tips but should check with Gustav beforehand whether the phrasing is legally fine. Also, we might need an entire section on tips which would be out of scope of this PR. If you want, I can open a follow-up PR to add a section about tips.
```
Depending on the added value and scope of the contribution, it might be eligible for receiving a tip from the KILT Treasury.
```

Reply:
At the moment, I think we can leave the TIP out. Since we haven't had a chance for Gustav to check and its fine since we can include it later once we have more documentation relevant to the TIP otherwise it feels slapped in there.